package.json

```diff
@@ -28,7 +28,7 @@
     "dev:full:database": "npm-run-all -p start:watch start:emulators start:database:replica",
     "dev:light": "npm-run-all -p start:watch start:database",
     "dev:site": "npm-run-all -p start:emulators dev:site:next",
-    "dev:site:next": "firebase functions:config:set runtime.env=development && next",
+    "dev:site:next": "firebase functions:config:set runtime.env=development && next -p 3035",
```

Comment:
explicitly setting the dev port to avoid clashing with other `nkowaokwu` projects